container/common/fsHandler.go

```diff
@@ -129,6 +129,10 @@ func (fh *realFsHandler) trackUsage() {
 			duration := time.Since(start)
 			if duration > longOp {
 				glog.V(2).Infof("du and find on following dirs took %v: %v", duration, []string{fh.rootfs, fh.extraDir})
+				// adapt longOp time so that message doesn't continue to print
```

Comment:
log something like the following:

"du and find on following dirs took %v: %v; will log in future if exceeds %v"